README.md

```diff
@@ -20,11 +20,11 @@ A tool to easily read auction house data with a loot logger, damage meter, dunge
 
 ### Prerequisites & Installation
 - You need **Windows 10** or higher
-- Install **.NET 6.0 Desktop Runtime** (v6.0.5 or higher) [here](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.5-windows-x64-installer) (To start the tool)
```

Comment:
You should not change this file, please undo these changes.
It's best to always use the latest dev branch and then make your changes.